web3.js/src/connection.js

```diff
@@ -1872,6 +1903,35 @@ export class Connection {
     return result.result;
   }
 
+  /**
+   * Fetch a list of all the confirmed signatures for transactions involving an address
+   * within a specified slot range. Max range allowed is 10,000 slots.
```

Comment:
This comment is copied from the old method

Reply:
😊